testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/observability/opentelemetry/BaseOpenTelemetryTest.java

```diff
@@ -19,44 +19,46 @@
 import org.jboss.shrinkwrap.api.asset.StringAsset;
 import org.jboss.shrinkwrap.api.spec.WebArchive;
 import org.wildfly.test.integration.observability.opentelemetry.application.JaxRsActivator;
+import org.wildfly.test.integration.observability.opentelemetry.application.MetricResource;
 import org.wildfly.test.integration.observability.opentelemetry.application.OtelService1;
 
 public abstract class BaseOpenTelemetryTest {
     private static final String WEB_XML
-            = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
-            + "<web-app xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns=\"http://java.sun.com/xml/ns/javaee\"\n"
-            + "         xsi:schemaLocation=\"http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd\"\n"
-            + "         metadata-complete=\"false\" version=\"3.0\">\n"
-            + "    <servlet-mapping>\n"
-            + "        <servlet-name>jakarta.ws.rs.core.Application</servlet-name>\n"
-            + "        <url-pattern>/*</url-pattern>\n"
-            + "    </servlet-mapping>"
-            + "</web-app>";
+        = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
```

Comment:
Not important but it is easier/tidier to include a class like this in your web app, which removes the need for this xml

```

import jakarta.ws.rs.ApplicationPath;
import jakarta.ws.rs.core.Application;

@ApplicationPath("/")
public class MyTestApplication extends Application {
}

```

Reply:
Agreed. There's even one in a subpackage there. This is just an old wart that I haven't gotten around to cleaning up, but I'll get that on the next push.